test/e2e-helm/e2e_helm_olm_test.go

```diff
@@ -63,6 +74,19 @@ var _ = Describe("Integrating Helm Projects with OLM", func() {
 			err = tc.Make("packagemanifests", "IMG="+tc.ImageName)
 			Expect(err).NotTo(HaveOccurred())
 
+			By("running basic scorecard tests")
+			var scorecardOutput v1alpha3.TestList
+			runScorecardCmd := exec.Command(tc.BinaryName, "scorecard", "bundle",
+				"--selector=suite=basic",
```

Comment:
I do not think that we should use selector by default.

Reply:
Looks the same to me? https://github.com/operator-framework/operator-sdk/blob/master/test/e2e/e2e_suite_test.go#L217-L220

Reply:
Ops. It is true . Closing that. 👍